challenges.md

```diff
@@ -1,68 +1,110 @@
 # About the Code Challenges
-We have a couple of code challenges designed to give you the opportunity to showcase your skills. We love to see creativity, originality and simplicity of design. All of our projects involve some creativity, research, interest in data, automation and of course amazing instructions. Please make sure to write the most incredible and informative README.md files you can imagine.
+Welcome to the Contrast Code Challenges project page. Code challenges are an important part of the recruiting and hiring process at Contrast. Every engineer/developer, designer or technologist that works at Contrast, full-time or intern submits a code challenge as part of the process to join the team.
 
-There are tons of amazing open and free data sets to play with. Here are a few for inspiration:
+The code challenge is designed to give candidates the ability to showcase their skills in a low stress, extended time model. The code challenge gives you a chance to demonstrate your creativity over a couple of days, rather than in a conference room as part of a technical interview. While we give you ample time, each code challenge is designed to take no more than 4 to 8 hours. We believe that technologists work best in the comfort of their own home, coffee shop or favorite place to code or design. 
```

Comment:
The code challenge is designed to give candidates the ability to showcase their skills in a low stress, extended time model rather than in a conference room as part of a technical interview. While we give you ample time, each code challenge is designed to take no more than 4 to 8 hours. We believe that technologists work best in the comfort of their own home, coffee shop or favorite place to code or design.